src/commands/Utility/dictionary.js

```diff
@@ -0,0 +1,32 @@
+const webdict = require('webdict');
+const richEmbed = require('discord.js').RichEmbed;
+
+exports.run = function (bot, msg, args) {
+    msg.edit(':arrows_counterclockwise:');
```

Comment:
Please remove this. Not only will the timing possible cause problems, but it will add latency to the response if you wait for the edit to complete.